vllm/benchmarks/throughput.py

```diff
@@ -246,12 +243,15 @@ async def run_vllm_async(
 def run_hf(
     requests: list[SampleRequest],
     model: str,
-    tokenizer: PreTrainedTokenizerBase,
+    tokenizer: TokenizerLike,
     n: int,
     max_batch_size: int,
     trust_remote_code: bool,
     disable_detokenize: bool = False,
 ) -> float:
+    assert isinstance(tokenizer, PreTrainedTokenizerBase), (
```

Comment:
this works out to be true since TokenizerLike only wraps an AutoTokenizer.from_pretrained object in [hf.py](https://github.com/vllm-project/vllm/blob/main/vllm/tokenizers/hf.py)